packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -148,3 +167,207 @@ contract OPContractsManager_InternalMethods_Test is Test {
         vm.assertEq(expected, actual);
     }
 }
+
```

Comment:
Should be a test that triggers `OnlyDelegateCall()`.

Reply:
done